extensions/ql-vscode/src/variant-analysis/run-remote-query.ts

```diff
@@ -262,7 +262,7 @@ async function copyExistingQueryPack(
   await fixPackFile(queryPackDir, packRelativePath);
 }
 
-async function findPackRoot(queryFile: string): Promise<string> {
+export async function findPackRoot(queryFile: string): Promise<string> {
```

Comment:
In a later PR I'll move this to [`ql.ts`](https://github.com/github/vscode-codeql/blob/9285b0272df7b7a17eeb907f2ec8e5f6f9404bfa/extensions/ql-vscode/src/common/ql.ts)